client/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-router-bootstrap": "^0.25.0",
```

Comment:
I don't think you need this library here, we don't use bootstrap in the project too, so this can be removed i think.